package.json

```diff
@@ -13,21 +13,20 @@
   },
   "scripts": {
     "prepare": "husky install",
-    "clean": "rm -rf ./build ./dist",
+    "clean": "rm -rf ./build",
     "lint": "eslint ./src",
     "format": "prettier -w ./src config-overrides.js",
     "tsc": "tsc",
-    "build": "DOTENV_CONFIG_PATH=.env.production GENERATE_SOURCEMAP=false react-app-rewired build",
-    "build:dev": "GENERATE_SOURCEMAP=false react-app-rewired build",
-    "watch": "node ./scripts/watch",
-    "start": "react-app-rewired start",
-    "test": "react-app-rewired test",
+    "build": "tsc && NODE_ENV=production webpack",
```

Comment:
we don't, I've just added this because otherwise we never do any type-checking in the build process since webpack is set to "transpile only"

Reply:
isn't there a flag we can pass to `webpack` ?

Reply:
we could, the issue is that webpack is instrumenting the module loader of node and I think that causes issues with the way things are type-checked, because when I "tell webpack" to do the type-checking it fails, I haven't solved that yet. It's the same with jest, it doesn't type-check either.